docs/001_develop/03_client-capabilities/008_interaction/slider.mdx

```diff
@@ -0,0 +1,275 @@
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax.
+
+### Attributes
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>readonly</td>
+      <td><code>boolean</code></td>
+      <td>The slider is for display only, and cannot be changed by the user.</td>
+<td>
+
+```typescript
+<rapid-slider readonly>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Similar to `readonly`, but component is also greyed out.</td>
+<td>
+
+```typescript
+<rapid-slider disabled>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>form</td>
+      <td><code>string</code></td>
+      <td>Associates this component to a form. Form `id` needs to be passed. If no Id informed, then it will be associated with the ancestor form.</td>
+<td>
+
+```typescript
+<rapid-slider form="myForm">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>max</td>
+      <td><code>number</code></td>
+      <td>Defines maximum number of the slider.</td>
+<td>
+
+```typescript
+<rapid-slider max=50>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>min</td>
+      <td><code>number</code></td>
+      <td>Defines minimum number of the slider.</td>
+<td>
+
+```typescript
+<rapid-slider min=10>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>step</td>
+      <td><code>number</code></td>
+      <td>Defines the value of each step of the slider.</td>
+<td>
+
+```typescript
+<rapid-slider step=5>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>value</td>
+      <td><code>number</code></td>
+      <td>Defines a value for the component when it is created.</td>
+<td>
+
+```typescript
+<rapid-slider value=20>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>orientation</td>
+      <td><code>string</code></td>
+      <td>Defines the orientation: `vertical` or `horizontal`.</td>
+<td>
+
+```typescript
+<rapid-slider orientation="horizontal">
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+### Properties
+This component doesn't have any properties which are not also controlled via attributes.
+
+### Slots
+
+| Slot | Use |
+|---|---|
+| checked-indicator	| The checked indicator. |
+| track |	The track of the slider. |
+| track-start	| The track-start visual indicator. |
+| thumb	| The slider thumb. |
+
+### Parts
+
+| Part | Description |
+|---|---|
+| control |	The element representing the visual radio control. |
+| label |	The label. |
+| positioning-region | The region used to position the elements of the slider. |
+| track | The region containing the track elements. |
+| track-start | The element wrapping the track start slot. |
+| thumb-container | The thumb container element which is programmatically positioned. |
+| thumb	| The thumb element. |
+
+### Events fired
+| Name    | Description                                 |
+|---------|---------------------------------------------|
+| @change | Fires the event when the slider component changes its state. |
```

Comment:
Let's put an example of using the change event here, [like this](https://github.com/genesiscommunitysuccess/docs/pull/1895/files#diff-de5bad40a087c6287b3752c04167e66df977b97ed8150a9d0566e108c975053dR213-R236)